cli/src/commands/start.rs

```diff
@@ -618,6 +625,45 @@ impl Start {
             false => self.rest.or_else(|| Some("0.0.0.0:3030".parse().unwrap())),
         };
 
+        // Initialize the storage mode.
+        let storage_mode = match &self.storage {
+            Some(path) => StorageMode::Custom(path.clone()),
+            None => match self.dev {
+                Some(id) => StorageMode::Development(id),
+                None => StorageMode::Production,
+            },
+        };
+
+        // Helper function to obtain the path to the JWT secret.
+        let jwt_secret_path = |network: u16, storage_mode: &StorageMode, address: &Address<N>| {
+            let mut jwt_secret_path = aleo_std::aleo_ledger_dir(network, storage_mode);
+            jwt_secret_path.push(format!("jwt_secret_{address}.txt"));
+            jwt_secret_path
+        };
```

Comment:
nit: it seems that this path is constructed just once, so a closure isn't really needed; it's not a blocker ofc